### PR TITLE
Closes #2271: fix hooks for github

### PR DESCRIPTION
**Pull request by sfrevel**
---
Release Notes:

```
fixed the commit hooks for github
```
 
### For the submitter:
- [ ] I updated the [documentation](https://taskana.atlassian.net/wiki/spaces/TAS/overview) and will supply links to the specific files
- [ ] I did not update the [documentation](https://taskana.atlassian.net/wiki/spaces/TAS/overview)
- [ ] I included a link to the [SonarCloud branch analysis](https://taskana.atlassian.net/wiki/spaces/TAS/pages/1019969636/SonarCloud+Integration)
- [ ] After integration of the PR, I added a description of changes on the [current release notes](https://taskana.atlassian.net/wiki/spaces/TAS/pages/1281392672/Current+Release+Notes+Taskana)
- [ ] I did not update the [current release notes](https://taskana.atlassian.net/wiki/spaces/TAS/pages/1281392672/Current+Release+Notes+Taskana)
- [ ] I put the ticket in review
- [ ] After integration of the pull request, I verified our [bluemix test environment](http://taskana.mybluemix.net/taskana) is not broken

### Verified by the reviewer:
- [ ] Commit message format → TSK-XXX: Your commit message.
- [ ] Submitter's update to [documentation](https://taskana.atlassian.net/wiki/spaces/TAS/overview) is sufficient
- [ ] SonarCloud analysis meets our standards
- [ ] Update of the [current release notes](https://taskana.atlassian.net/wiki/spaces/TAS/pages/1281392672/Current+Release+Notes+Taskana) reflects changes
- [ ] PR fulfills the ticket
- [ ] Edge cases and unwanted side effects are tested
- [ ] Readability

### CODE REVIEW

File: .github/pull_request_template.md [hunk 1]
```diff
@@ -16,7 +16,7 @@ Release Notes:
 - [ ] After integration of the pull request, I verified our [bluemix test environment](http://taskana.mybluemix.net/taskana) is not broken
 
 ### Verified by the reviewer:
-- [ ] Commit message format → TSK-XXX: Your commit message.
+- [ ] Commit message format → (Close|Closes|Closed|Fix|Fixes|Fixed|Resolve|Resolves|Resolved) #<Issue Number>: Your commit message. i.e. Closes #<2271>: Your commit message.
```
[the remark this answers is not in the document]
Comment (arolfes) [reply .github/pull_request_template.md:19]: I said to use all which are supported by github for the commit hook.

For the template only one is enough.
Currently we use only Closes

@sfrevel: can you change the template

**Comment by arolfes** [comment]
Please change your commit message to

`Closes #2271: fix hooks for github`

**Comment by sfrevel** [comment]
So we don't want <> ? I misunderstood this. But then I have to remove thes in the template and the commit-msg examples 
also...

**Comment by arolfes** [comment]
I like the colon :-)

@mustaphazorgati : What do you say?

**Comment by arolfes** [comment]
do it without colon

**Review by arolfes — APPROVED** [review]

**Pull request merged**